### ISSUE TITLE
Settings editor: Folder action item not handling keyup properly

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#141977]
1. Open a multi-root workspace file.
2. Navigate to the Settings editor.
3. Press tab until focus reaches the scope selector widget.
4. Press the right arrow key until focus reaches the folder option. The folder option should be to the right of the user and workspace tabs.
5. Press the Enter or Space key.
6. :bug: There is supposed to be a dropdown, but it doesn't show up.

Example screenshot showing what the folder option looks like when focused:
![Screenshot](https://user-images.githubusercontent.com/7199958/152077574-b8199bbe-56df-444e-b558-3a839c99f85b.PNG)

**Comment by rchiodo** [comment]
/verified

The popup works when hitting the enter key:

![image](https://user-images.githubusercontent.com/19672699/155595031-92129824-67b1-4f00-8ada-640088510037.png)
